src/Handling/SpellCheckMiddleware.php

```diff
@@ -2,15 +2,15 @@
 
 namespace SilverStripe\SpellCheck\Handling;
 
-use SilverStripe\Control\Director;
 use SilverStripe\Control\HTTPRequest;
 use SilverStripe\Control\Middleware\HTTPMiddleware;
 use SilverStripe\Core\Config\Configurable;
-use SilverStripe\Forms\HTMLEditor\HTMLEditorConfig;
 use SilverStripe\Forms\HTMLEditor\TinyMCEConfig;
 use SilverStripe\i18n\i18n;
-use SilverStripe\Security\SecurityToken;
 
+/**
+ * @deprecated 2.0..3.0 Use SpellCheckAdminExtension instead
```

Comment:
If we're deprecating we need to bump the minor version - can we switch this to the master branch as 2.1? I can update the recipe that this lives in for the new version in CWP 2.1.

Reply:
ok if you want. I'll let you merge to the branch you think is best.